index.html

```diff
@@ -0,0 +1,269 @@
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+            <div class="showcase">
+                <div class="showcase-image">
+                   <img src="images/Img Placeholder (4).png" alt="">
+                </div>
+
+                <div class="showcase-detail">
+                  <h1>Profesional Art Printing Data</h1>
+                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                  <ul>
+                      <li>HTML</li>
+                      <li>Bootstrap</li>
+                      <li>Ruby</li>
+                  </ul>
+              </div>
+                <button>See project</button>
+            </div>
+        </div>
+
+   </section>
+
+   <!--About Me Section-->
+
+   <section class="about-me">
+       <div class="my-details">
+          <h1>About me</h1>
```

Comment:
- Kindly you will have one `h1` element per page, so use other heading elements for this, I think `h2` would be better for better semantic arrangement 👍